pkg/storaged/storage-anaconda.scss

```diff
@@ -19,3 +19,13 @@
     --pf-v6-c-card--BorderColor: transparent;
   }
 }
+
+body:has(.storage-anaconda) {
+  // Allow scrolling if a dropdown menu gets positioned outside the bounds of the viewport. This
+  // might happen when multiple partitions are created resolutions on 1080p or lower resolutions
+  // (see https://bugzilla.redhat.com/show_bug.cgi?id=2388785)
+  overflow-y: auto !important;
+  // containers like pf-v6-c-page won't expand with the viewport on the edge-case described above,
+  // so we force all background to match the default
+  background: var(--pf-t--global--background--color--primary--default);
```

Comment:
without setting this change, it would look like the following

<img width="1350" height="820" alt="installer-4272-overflow-without-body-background" src="https://github.com/user-attachments/assets/21d61e49-6e42-4f02-a93a-fe3ff4ce0cde" />

Reply:
Normally, a dropdown will automatically open upwards instead of downwards if there is not enough space below it. But maybe in your case there isn't even enough space above it?

<img width="383" height="484" alt="image" src="https://github.com/user-attachments/assets/413c8953-a95d-4896-8e01-70c9761f8488" />

Cockpit without Anaconda has the same problem, actually, when it is really shallow:

<img width="383" height="484" alt="image" src="https://github.com/user-attachments/assets/cc5e9145-a7d9-4b72-be06-eba1725addb7" />

So, I think we should apply your fix in general, not just for Anaconda mode. Also, isn't this a PatternFly bug? @Venefilyn?